### ISSUE TITLE
Characters limit

### ISSUE THREAD

**Issue by Sheendrah** [owochle/ChatPimper#1]
Hello, ty for your work ! it is incredible ! But... 
The character limitation is problematic because although I do not exceed 256 characters, the message does not want to be sent if I do not activate splitting. Which is disturbing on a multiplayer server (could be considered spam in addition to not being aesthetic at all :D)
As soon as I am at 29 characters (space included obviously) I get an error message:
_"The message is too long (more than 256 characters)"_

Could you please fix this problem? (on a solo game no problem, just in multiplayer) Thank you ♥

PS: When i disabled gradient option, no more issues with the length of my messages 🤔

**Comment by owochle** [comment]
Yes, it's because the way the gradient works is by prefix **every** character by `&xrrggbb` so, it sends the character count to the roof. Sadly, there isn't much I can do, this limit is kinda hard coded into minecraft.

**Comment by Sheendrah** [comment]
Ok i see, but why does it work on my solo game, but not on multi ? Does the game work same in this two cases ?

**Comment by owochle** [comment]
In solo, messages are not sent anywhere, they are directly printed on screen in chat, so there is no such limitation.
